src/features/tweaks/highlight_contributed_content.js

```diff
@@ -1,7 +1,7 @@
 import { buildStyle } from '../../utils/interface.js';
 
 export const styleElement = buildStyle(`
-[data-is-contributed-content="true"] {
-  display: flow-root; background-color: rgb(var(--follow));
+[data-is-contributed-content="true"]:not([data-xkit-themed]) {
+  display: flow-root; background-color: rgb(var(--deprecated-accent), 0.1);
```

Comment:
when I last looked at this (in July 2022, ouch), I calculated that a _13_%-opacity accent matched the luminance of `--follow`
```suggestion
  display: flow-root; background-color: rgb(var(--deprecated-accent), 0.13);
```